web/src/components/OutageAlertBar.tsx

```diff
@@ -1,15 +1,30 @@
 import { AuthContext } from "@/contexts/authContext";
 import { IsAuthenticated } from "@/lib/auth/AuthContext";
-import { ReactElement, useContext } from "react";
-
-type OutageAlertType = "ALL" | "UNREGISTERED_ONLY" | "LOGGED_IN_ONLY" | undefined;
+import { OutageAlertType, OutageConfig } from "@/lib/types/types";
+import { useMountEffect } from "@/lib/utils/helpers";
+import axios from "axios";
+import { ReactElement, useContext, useState } from "react";
 
 export const OutageAlertBar = (): ReactElement => {
   const { state } = useContext(AuthContext);
 
-  const isEnabled = process.env.FEATURE_ENABLE_OUTAGE_ALERT_BAR === "true";
-  const message = process.env.OUTAGE_ALERT_MESSAGE;
-  const alertType = process.env.OUTAGE_ALERT_TYPE as OutageAlertType;
+  const [outageConfig, setOutageConfig] = useState<OutageConfig>({
+    FEATURE_ENABLE_OUTAGE_ALERT_BAR: false,
+    OUTAGE_ALERT_MESSAGE: "",
+    OUTAGE_ALERT_TYPE: undefined,
+  });
+
+  const isEnabled = outageConfig.FEATURE_ENABLE_OUTAGE_ALERT_BAR;
+  const message = outageConfig.OUTAGE_ALERT_MESSAGE;
+  const alertType = outageConfig.OUTAGE_ALERT_TYPE as OutageAlertType;
+  const urlToFetchFrom = process.env.OUTAGE_ALERT_CONFIG_URL || "";
+
+  useMountEffect(() => {
+    if (!urlToFetchFrom) return;
+    axios.get(urlToFetchFrom).then((response) => {
```

Comment:
[pebble] I think we need to handle for unsuccessful calls. If for some reason we can't resolve this wouldn't a error show up in the UI?

Reply:
Literally the only reason I'm thinking about this is because GH had downtime earlier today.

Reply:
@jphechter I didn't think so? The default value is set on lines 11-15 and defaults to NOT showing the Outage Bar. So there's no way that the FEATURE_ENABLE_OUTAGE_ALERT_BAR value would be set to true unless the call succeeds, right? A failed call should just fail silently

Reply:
I wasn't thinking the alert bar would show up. I was thinking about the React errors, but that won't show up in prod any how. Never mind.

<img width="685" alt="Screenshot 2023-07-21 at 3 47 09 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22647707/8e4f07be-b491-47ec-b010-6a3f8d18ff75">

Reply:
Oh hm. That's fair, I can throw in a catch